docs/guides/lender.md

```diff
@@ -1 +1,916 @@
-# Lender
\ No newline at end of file
+# Lender
+
+In this guide, you’ll learn how to build a simple defi dApp in under an hour using the [Código Platform](https://studio.codigo.ai). We will be building a dApp that will allow informal lenders to lend money.
+
+Código is an AI-powered code Generation Platform for blockchain developers and web3 teams that saves development time and increases the code's security.
+
+To get started, create an account by accessing [Código Studio](https://studio.codigo.ai) 
+
+## Create an account
+When entering ![](https://studio.codigo.ai) you will be received with the following screen. You can access Código Studio using your GitHub account or by creating a new account with any email address:
+
+![Código Studio Sign Up](../../static/img/sign_codigo_studio.png)
+![Código Studio Register](../../static/img/register_codigo_studio.png)
+
+Código Studio is a web-based IDE that allows you to develop dApp using Solana and the Código Platform. The web-based IDE comes with all the required components to build Solana dApp i.e. Solana Development suite tool, Anchor, Código CLI, Node.js, Rust, and more
+
+:::info
+The first time you create an account, provisioning your environment can take **3 to 5 minutes**.
+:::
+
+## Create & implement the CIDL for a simple DeFi
+
+After creating your account and successfully accessing the environment we can start writing the CIDL. Create a new file named `cidl.yaml` from your file explorer. 
```

Comment:
change the file name to `main.cidl`, now with the extension we will have support for `.cidl`